build-time-analytics/src/test/resources/anonymous-id.txt

```diff
@@ -0,0 +1 @@
+2c5149b1-3ad4-4d5e-b287-1938c93aad55
```

Comment:
Is this autogenerated?

Reply:
It is. Thanks for pointing it out, this is not necessary here. I'll drop it.